ui/components/app/hold-to-reveal/hold-to-reveal.js

```diff
@@ -0,0 +1,87 @@
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { CircularProgress } from '@material-ui/core';
+import Button from '../../ui/button';
+import { I18nContext } from '../../../contexts/i18n';
+import Box from '../../ui/box/box';
+
+export default function HoldToReveal({
+  buttonText,
+  timeToHold,
```

Comment:
Let's set a default here, maybe like 1 second?  `timeToHold = 1`

Reply:
On second thought, I think we should also change the parameter name.  `time` signals milliseconds to me, so we should change to `secondsToHold`.